### PR TITLE
updated login and signup page

### PR DESCRIPTION
**Pull request by tushitshukla03**

### CODE REVIEW

File: src/assets/Group 1f1.png
Comment (Priyansh61) [inline src/assets/Group 1f1.png:1]: We dont need this ig

File: src/assets/Group 2f2.png
Comment (Priyansh61) [inline src/assets/Group 2f2.png:1]: This is also not needed.

File: src/assets/Group 4f3.png
Comment (Priyansh61) [inline src/assets/Group 4f3.png:1]: Same here

**Review by Priyansh61 — CHANGES_REQUESTED** [review]
image


1. I guess the card is not in the center. Could you please check that ?
2. Remove the naming login form just keep it login 
3. Same, just keep the other signup
4. Keep the login/signup with google at top then add or for other options

**Comment by tushitshukla03** [comment]
> image
> 
> 1. I guess the card is not in the center. Could you please check that ?
> 2. Remove the naming login form just keep it login
> 3. Same, just keep the other signup
> 4. Keep the login/signup with google at top then add or for other options

please check now

**Pull request merged**